raiden/raiden_service.py

```diff
@@ -227,13 +227,17 @@ def update_services_from_balance_proof(
     raiden: "RaidenService",
     chain_state: "ChainState",
     balance_proof: Union[BalanceProofSignedState, BalanceProofUnsignedState],
+    monitoring_service_contract_address: Address,
```

Comment:
the MSC address is stored in the `raiden` instance, so you could use it from there.

Reply:
Good point!